.github/workflows/kind.yml

```diff
@@ -197,7 +197,7 @@ jobs:
     - name: Upload Test Reports
       uses: actions/upload-artifact@v3
       with:
-        name: fw-kind-report-${{ matrix.kind }}-${{ matrix.deployOnHub }}${{ matrix.hosted == 'hosted' && '-hosted' }}
+        name: fw-kind-report-${{ matrix.kind }}-${{ matrix.deployOnHub }}${{ matrix.hosted == 'hosted' && '-hosted' || '' }}
```

Comment:
Can we use like this  matrix.hosted == 'hosted' ? '-hosted' : ''  (2. Ternary Operator or condition operator)

Reply:
This is the closest I think we can get to a real ternary operator in github actions syntax :/ https://docs.github.com/en/actions/learn-github-actions/expressions#example

Reply:
Thanks!